dio/lib/src/headers.dart

```diff
@@ -91,14 +91,16 @@ class Headers {
   /// header. The header name passed in [:name:] will be all lower
   /// case.
   void forEach(HeaderForEachCallback f) {
-    _map.keys.forEach((key) => f(key, this[key]!));
+    for (final key in _map.keys) {
+      f(key, this[key]!);
+    }
   }
 
   @override
   String toString() {
```

Comment:
Maybe one can use `string.join` here?

Reply:
Hmm yea, there is a test for this so I can try that later.

Reply:
I have no strong opinion on this, I just thought that it may be easier to read.